src/components/filters/FilterList/index.jsx

```diff
@@ -85,47 +85,64 @@ class FilterList extends React.Component {
     // Takes in parent component's filterStatus or self state's filterStatus
     const filterStatus = this.props.filterStatus
       ? this.props.filterStatus : this.state.filterStatus;
-
+    const filterSectionToShow = [];
+    this.props.sections.forEach((section, index) => {
```

Comment:
We could change this from a `forEach` to a `map` like...

```
const filterSectionToShow = this.props.sections.map((section, index) => {
...
return (
  <FilterSection
...
```

Unless some side-effect logic is happening in the loop that I'm not seeing 🔍

Reply:
i will revisit, map will not work due to below if statement, but reduce will probably work

Reply:
do you have any thoughts? i think forEach may be the best way to do this

Reply:
Ah, yeah, I see. `forEach` works! Another way to do it would be to return `null` and filter, but at that point the `forEach` makes more sense. Looks good as is!